examples/json_consumer_example/json_consumer_example.go

```diff
@@ -0,0 +1,134 @@
+// Example function-based high-level Apache Kafka consumer
+package main
+
+/**
+ * Copyright 2022 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// consumer_example implements a consumer using the non-channel Poll() API
+// to retrieve messages and events.
+
+import (
+	"fmt"
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/confluentinc/confluent-kafka-go/schemaregistry"
+	"github.com/confluentinc/confluent-kafka-go/schemaregistry/serde"
+	"github.com/confluentinc/confluent-kafka-go/schemaregistry/serde/jsonschema"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func main() {
+
+	if len(os.Args) < 5 {
+		fmt.Fprintf(os.Stderr, "Usage: %s <schema-registry> <broker> <group> <topics..>\n",
+			os.Args[0])
+		os.Exit(1)
+	}
+
+	url := os.Args[1]
+	broker := os.Args[2]
+	group := os.Args[3]
+	topics := os.Args[4:]
+	sigchan := make(chan os.Signal, 1)
+	signal.Notify(sigchan, syscall.SIGINT, syscall.SIGTERM)
+
+	c, err := kafka.NewConsumer(&kafka.ConfigMap{
+		"bootstrap.servers": broker,
+		// Avoid connecting to IPv6 brokers:
+		// This is needed for the ErrAllBrokersDown show-case below
+		// when using localhost brokers on OSX, since the OSX resolver
+		// will return the IPv6 addresses first.
+		// You typically don't need to specify this configuration property.
+		"broker.address.family": "v4",
+		"group.id":              group,
+		"session.timeout.ms":    6000,
+		"auto.offset.reset":     "earliest"})
+
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to create consumer: %s\n", err)
+		os.Exit(1)
+	}
+
+	fmt.Printf("Created Consumer %v\n", c)
+
+	client, err := schemaregistry.NewClient(schemaregistry.NewConfig(url))
+
+	if err != nil {
+		fmt.Printf("Failed to create schema registry client: %s\n", err)
+		os.Exit(1)
+	}
+
+	deser, err := jsonschema.NewDeserializer(client, serde.NewConfig(), serde.ValueSerde, serde.DisableValidation)
```

Comment:
wouldn't `DisableValidation` be better as a config property?

Reply:
Added config prop `EnableValidation`